lib/Interpreter/CppInterOp.cpp

```diff
@@ -2415,7 +2415,7 @@ namespace Cpp {
     } // namespace
       // End of JitCall Helper Functions
 
-  JitCall MakeFunctionCallable(TCppConstFunction_t func) {
+  DllExport JitCall MakeFunctionCallable(TCppConstFunction_t func) {
```

Comment:
We do not need to export things outside of the public API that's defined in the header file.